cly/utils.py

```diff
@@ -47,7 +47,7 @@ def parse_arguments(arguments: Union[str, List[str]]) -> str:
 
 def get_output(
     arguments: Union[str, List[str]], directory: Optional[Path] = None
-) -> subprocess.CompletedProcess:  # type: ignore
+) -> "subprocess.CompletedProcess[str]":
```

Comment:
https://stackoverflow.com/questions/68089860/completedprocess-is-a-generic-for-mypy